profile/README.md

```diff
@@ -0,0 +1,30 @@
+<p align="center">
+    <img width="400px" src="https://raw.githubusercontent.com/FluxML/Flux.jl/master/docs/src/assets/logo.png"/>
+</p>
+
+# The Elegant Machine Learning Stack
+
+Flux is a 100% pure-Julia stack and provides lightweight abstractions on top of Julia's native GPU and AD support. It makes the easy things easy while remaining fully hackable.
+
+## Features
+
+Flux has features that sets it apart among ML systems
+
+### Compiled Eager Code
+Flux provides a single, intuitive way to define models, just like mathematical notation. Julia transparently [compiles your code](https://julialang.org/blog/2018/12/ml-language-compiler), optimising and fusing kernels for the GPU, for the best performance.
+
+### Differentiable Programming
+Existing Julia libraries are differentiable and can be incorporated directly into Flux models. Cutting edge models such as [Neural ODEs](https://julialang.org/blog/2019/01/fluxdiffeq) are first class, and [Zygote](https://github.com/FluxML/Zygote.jl) enables overhead-free gradients.
+
+### First-class GPU support
+GPU kernels can be written directly in Julia via [CUDA.jl](https://github.com/JuliaGPU/CUDA.jl). Flux is uniquely hackable and any part can be tweaked, from GPU code to custom gradients and layers.
+
+### The Model Zoo
+A [rich collection](https://github.com/FluxML/model-zoo) of Flux scripts to learn from, or tweak to your own data. Trained Flux models can be used from [TextAnalysis](https://github.com/JuliaText/TextAnalysis.jl) or [Metalhead](https://github.com/FluxML/Metalhead.jl).
```

Comment:
At some point we thought about toning down these claims, but I don't remember the specifics. @darsnack do you recall?

Reply:
That looks good. If we broaden the headline to be more like "Model Ecosystem", the inclusion of Metalhead would make sense as well. I would also nominate Transformers.jl and maybe Flux3D.

Reply:
In https://github.com/FluxML/fluxml.github.io/pull/145 I simply removed the line. It's just misleading and drives people to the zoo where the get disappointed. My plan was to put it back in once we port the zoo examples and website tutorials into *real* tutorials that stay up to date in the docs.

I would remove any reference to the zoo here too.

Reply:
Found it - https://github.com/FluxML/Flux.jl/pull/2069/files#diff-36c852523b6a7513a10b8a6bae21f1037eaa27484efc9042f75bbda46d9f529bR6-R7. Maybe I should add in these  2 points too?

Reply:
Where does this readme show up?

I wonder if it shouldn't say almost nothing, and direct people to the website / the main Flux repository. 

Having so many different blurbs in different places seems a bit confusing, and harder to keep up to date.

Reply:
Fair point. This will show up like this - https://github.com/pybamm-team.